### PR TITLE
hw1

### PR DESCRIPTION
**Pull request by i-baumann**
adding hw1 file, updating requirements, extending flake8 character limit

### CODE REVIEW

File: hw/hw1-fisher.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+iris = pd.read_csv(
+    url, names=["sepal_length", "sepal_width", "petal_length", "petal_width", "class"]
+)
+
+# Descriptive table (pandas)
+print("********** Iris Data Descriptive Stats **********")
+rows = ["mean", "min", "25%", "50%", "75%", "max"]
+pandas_table = np.round(iris.describe(), 2).loc[rows]
+print(pandas_table)
+
```
Comment (colinanderson10) [inline hw/hw1-fisher.py:23]: Maybe add numpy descriptives to at least show you know the code

**Review by colinanderson10 — COMMENTED** [review]
Everything works perfectly when running, the only suggestion I have is to add some numpy descriptives in addition to the simpler pandas describe method

File: hw/hw1-fisher.py [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+import pandas as pd
+import plotly.express as px
+import plotly.io as pio
+from sklearn.ensemble import AdaBoostClassifier, RandomForestClassifier
+from sklearn.neighbors import KNeighborsClassifier
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import Normalizer
+
+# Snag data
+url = "https://archive.ics.uci.edu/ml/machine-learning-databases/iris/iris.data"
```
Comment (dafrenchyman) [inline hw/hw1-fisher.py:12]: Would have been cleaner with a main.

**Review by dafrenchyman — APPROVED** [review]
The code would have been better organized if there was a main function. The function that was inline would have been separated out and been cleaner too. Other than that, great work. Clean, easy to follow, etc…Great job.

**Pull request merged**